### PR TITLE
Performed literary review and added/fixed links for "Minimal Orleans Application"

### PR DESCRIPTION
**Pull request by raghav710**
* Added link for getting started with Orleans
* Fixed typos and grammar
* Corrected link for Running in a stand alone silo

### CODE REVIEW

File: src/Tutorials/Minimal-Orleans-Application.md [hunk 1]
```diff
@@ -8,20 +8,20 @@ title: Minimal Orleans Application
...
 
 You can find the final source code in `Samples\Tutorial.Minimal` folder ([see latest version online](https://github.com/dotnet/orleans/tree/master/Samples/Tutorial.Minimal)).
 
 ## Project Setup
 
 
-For this tutorial we’re going to need to create 3 projects.  A Library that contains the interfaces (Communication interfaces), a library that contains the implementation (called Grains), and a simple console application that will Host our Silo.  For further understanding of this terminology please see the document Getting Started with Orleans.
+For this tutorial we’re going to need to create 3 projects.  A Library that contains the interfaces (Communication interfaces), a library that contains the implementation (called Grains), and a simple console application that will Host our Silo.  For further understanding of this terminology please see the document [Getting Started with Orleans](https://dotnet.github.io/orleans/Tutorials/My-First-Orleans-Application.html).
```
Comment (jdom) [inline src/Tutorials/Minimal-Orleans-Application.md]: you should be able to add a relative link to the markdown file instead, which will then be converted correctly depending on where the docs are hosted.
In this case it should be `[Getting Started with Orleans](My-First-Orleans-Application.md)

File: src/Tutorials/Minimal-Orleans-Application.md [hunk 2]
```diff
@@ -224,13 +224,13 @@ Within the appropriate bin directory (Debug/Release) there will be a number of l
...
+    - Solution: Set the property "Copy to Output Directory" to "Copy if newer", to set it, right click on the configuration file, then click on properties.
     ```
 
 ## Further Reading
 
  - [List of Orleans Packages](http://dotnet.github.io/orleans/NuGets)
  - [Orleans Configuration Guide](http://dotnet.github.io/orleans/Orleans-Configuration-Guide/)
  - [Orleans Best Practices](http://research.microsoft.com/pubs/244727/Orleans%20Best%20Practices.pdf)
- - [Running in a Stand Alone Silo](http://dotnet.github.io/orleans/Step-by-step-Tutorials/Running-in-a-Stand-alone-Silo)
+ - [Running in a Stand Alone Silo](https://dotnet.github.io/orleans/Tutorials/Running-in-a-Stand-alone-Silo.html)
```
Comment (jdom) [inline src/Tutorials/Minimal-Orleans-Application.md]: Ditto on the relative link to the md file

Comment (raghav710) [inline src/Tutorials/Minimal-Orleans-Application.md]: Thanks a lot for reviewing this :+1: It's always a pleasure contributing to open source! I'll make the changes as soon as possible

**Comment by raghav710** [comment]
@jdom incorporated the review comments. And yes, its the diversity of the team that makes open source better, I am happy to pitch in where required, just like the others.

**Comment by jdom** [comment]
Once again, thanks @raghav710!

**Pull request merged**